tests/e2e/vsphere_volume_expansion.go

```diff
@@ -199,8 +238,440 @@ var _ = ginkgo.Describe("Volume Expansion Test", func() {
+func convertBytesToMb(size resource.Quantity) int64 {
+	actualSize, _ := size.AsInt64()
+	actualSize = actualSize / (1024 * 1024)
+	return actualSize
+}
+
+//createStaticPVC this method creates static PVC
+func createStaticPVC(f *framework.Framework, client clientset.Interface, namespace string, defaultDatastore *object.Datastore, pandoraSyncWaitTime int) (string, *v1.PersistentVolumeClaim, *v1.PersistentVolume, *storagev1.StorageClass) {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	curtime := time.Now().Unix()
+	randomValue := rand.Int()
+	curtimeinstring := strconv.FormatInt(curtime, 10)
+	val := strconv.FormatInt(int64(randomValue), 10)
+	val = string(val[1:3])
+	curtimestring := strconv.FormatInt(curtime, 10)
+	pvcName := "pvc-" + curtimestring + val
+
+	sc, err := createStorageClass(client, nil, nil, "", "", true, "")
+	gomega.Expect(err).NotTo(gomega.HaveOccurred())
+	framework.Logf("Storage Class Name :%s", sc.Name)
+
+	ginkgo.By("create resource quota")
+	createResourceQuota(client, namespace, rqLimit, sc.Name)
+
+	ginkgo.By("Creating FCD Disk")
+	fcdID, err := e2eVSphere.createFCD(ctx, "BasicStaticFCD"+curtimeinstring, diskSizeInMb, defaultDatastore.Reference())
+	gomega.Expect(err).NotTo(gomega.HaveOccurred())
+	framework.Logf("FCD ID :", fcdID)
+
+	ginkgo.By(fmt.Sprintf("Sleeping for %v seconds to allow newly created FCD:%s to sync with pandora", pandoraSyncWaitTime, fcdID))
+	time.Sleep(time.Duration(pandoraSyncWaitTime) * time.Second)
+
+	// Creating label for PV.
+	// PVC will use this label as Selector to find PV
+	staticPVLabels := make(map[string]string)
+	staticPVLabels["fcd-id"] = fcdID
+
+	ginkgo.By("Creating PV")
+	pv := getPersistentVolumeSpecWithStorageclass(fcdID, v1.PersistentVolumeReclaimDelete, sc.Name, nil)
+	pv, err = client.CoreV1().PersistentVolumes().Create(ctx, pv, metav1.CreateOptions{})
+	gomega.Expect(err).NotTo(gomega.HaveOccurred())
+	pvName := pv.GetName()
+
+	ginkgo.By("Creating PVC")
+	pvc := getPVCSpecWithPVandStorageClass(pvcName, namespace, nil, pvName, sc.Name)
+	pvc, err = client.CoreV1().PersistentVolumeClaims(namespace).Create(ctx, pvc, metav1.CreateOptions{})
+	gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+	ginkgo.By("Waiting for claim to be in bound phase")
+	err = fpv.WaitForPersistentVolumeClaimPhase(v1.ClaimBound, client, namespace, pvc.Name, framework.Poll, framework.ClaimProvisionTimeout)
+	gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+	ginkgo.By("Verifying CNS entry is present in cache")
+	_, err = e2eVSphere.queryCNSVolumeWithResult(pv.Spec.CSI.VolumeHandle)
+	gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+	volHandle := pv.Spec.CSI.VolumeHandle
+
+	// Wait for PV and PVC to Bind
+	framework.ExpectNoError(fpv.WaitOnPVandPVC(client, namespace, pv, pvc))
+
+	return volHandle, pvc, pv, sc
+}
+
+//createStorageClassAndPVC creates storageClass with allowVolumeExpansion set to true and Creates PVC waits till PV, PVC are in bound
```

Comment:
It is easy to get confused with this method and `createPVCAndStorageClass`, please give it a better name